data/Dockerfiles/dockerapi/server.py

```diff
@@ -206,7 +206,7 @@ def container_post__exec__system__fts_rescan(self, container_id):
   def container_post__exec__system__df(self, container_id):
     if 'dir' in request.json:
       for container in docker_client.containers.list(filters={"id": container_id}):
-        df_return = container.exec_run(["/bin/bash", "-c", "/bin/df -H '" + request.json['dir'].replace("'", "'\\''") + "' | /usr/bin/tail -n1 | /usr/bin/tr -s [:blank:] | /usr/bin/tr ' ' ','"], user='nobody')
+        df_return = container.exec_run(["/bin/bash", "-c", "/bin/df '" + request.json['dir'].replace("'", "'\\''") + "' | /usr/bin/tail -n1 | /usr/bin/tr -s [:blank:] | /usr/bin/tr ' ' ','"], user='nobody')
```

Comment:
@andryyy We need a new build of dockerapi, as I removed the -H.

Reply:
I did locally, but the view is broken anyway. 

My VM has 100G, the other stuff is wrong, too.